BuildingSync.xsd

```diff
@@ -424,6 +424,13 @@
                         </xs:sequence>
                       </xs:complexType>
                     </xs:element>
+                    <xs:element name="Files" minOccurs="0">
```

Comment:
`auc:Files` elements should be added at every premises level and to every system type.

Reply:
That works too.

Reply:
I'll implement this soon.